pkg/cmd/workspace/code.go

```diff
@@ -195,9 +197,20 @@ func openBrowserIDE(activeProfile config.Profile, workspaceId string, projectNam
 		}
 	}
 
-	view_util.RenderInfoMessageBold(fmt.Sprintf("Forwarded %s IDE port to %d.\nOpening browser...", projectName, *browserPort))
+	ideURL := fmt.Sprintf("http://localhost:%d", *browserPort)
+	// Wait for the port to be ready
+	for{
+		if checkPortReady(*browserPort) {
+			break
+		}
+		time.Sleep(500 * time.Millisecond)
+	}
+	
+	view_util.RenderInfoMessageBold(fmt.Sprintf("Forwarded %s IDE port to %d.\nOpening browser...\nPlease click the following link to open your web browser: %s", projectName, *browserPort, ideURL))
```

Comment:
Please change the message to:
```
Forwarded {PROJECT} IDE port to {IDE_URL}.
Opening browser...
```

The last sentence is unnecessary

Reply:
fixed